cloud_node:
  ros__parameters:
    min_range: 0.3
    max_distance: 200.0
    frame_id: lslidar
    use_gps_ts: false
    topic_name: lslidar_point_cloud

    
lslidar_node:
  ros__parameters:
    device_ip: 192.168.1.200
    msop_port: 2368
    difop_port: 2369
    frame_id: lslidar
    add_multicast: false
    group_ip: 224.1.1.2
    rpm: 600.0
    return_mode: 1
    time_synchronization: false
    model: LSC16
    output_packets_topic: lslidar_packet

